### PR TITLE
Astute CLI (remove unused code and examples)

### PR DESCRIPTION
**Pull request by warpc**
Big cleaning:
- update README;
- remove old examples for deploy (format was changed);
- remove deploy from CLI.

### CODE REVIEW

File: bin/astute [hunk 1]
```diff
@@ -104,22 +101,8 @@ def console_provision(orchestrator, reporter, environment)
...
-        res = analize_deploy orchestrator.deploy(reporter, environment['task_uuid'], environment['nodes'], environment['attributes'])
-      end
-      res
-    end
+  exit console_provision(orchestrator, reporter, environment) if opts[:command] == :provision
 rescue => e
   report_and_exit(e, opts[:verbose])
-end
-exit result
+end
```
Comment (yoojinl) [inline bin/astute]: new line

**Comment by yoojinl** [comment]
retest this please

**Comment by yoojinl** [comment]
++

**Comment by warpc** [comment]
retest this please

**Pull request merged**